core/lib/dal/migrations/20231013163109_create_snapshots_table.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE snapshots
+(
+    l1_batch_number  BIGINT    NOT NULL PRIMARY KEY,
+    miniblock_number BIGINT    NOT NULL,
```

Comment:
Let's discuss if we need to savae both l1 batch number and miniblock number in the DB. This is denormalization because you can get the l1 batch number -> miniblock number from `miniblocks` table. 

Although `miniblocks` might be empty until we apply this snapshot. What do we use the `miniblock_number` field for?

Reply:
I added it mostly out of convenience, its useful to pass miniblock number in snapshot metadata, but it definitely does not need to be stored in DB, corrected